bindings/python/src/file.rs

```diff
@@ -185,7 +192,7 @@ impl File {
     }
 
     pub fn __exit__(&mut self, _exc_type: PyObject, _exc_value: PyObject, _traceback: PyObject) {
-        self.0 = FileState::Closed;
+        let _ = self.close();
```

Comment:
Hi, @messense @Zheaoli is there a way for use to return error during exist context manager?